polar-core/src/polar.lalrpop

```diff
@@ -62,6 +62,7 @@ extern {
         "or" => lexer::Token::Or,           // or
         "not" => lexer::Token::Not,         // not
         "matches" => lexer::Token::Matches, // matches
+        "type" => lexer::Token::Type        // type
```

Comment:
Should we call this "template" instead of "type" if we're referring to them as templates?

Reply:
I think I'm going to standardize on "rule prototype." "Template" is left over from a hackathon project.